README.md

```diff
@@ -4,59 +4,75 @@
 [![Coveralls Status](https://img.shields.io/coveralls/FAForever/server/develop.svg)](https://coveralls.io/github/FAForever/server)
 [![semver](https://img.shields.io/badge/license-GPLv3-blue)](license.txt)
 
-This is the source code for the [Forged Alliance Forever](http://www.faforever.com/) server.
+This is the source code for the
+[Forged Alliance Forever](https://www.faforever.com/) server.
```

Comment:
lobby server, not server (I guess many people get confused by this)

Reply:
Yea, eventually I might add a section about what the code actually does. Hosting games, matchmaking, player status, etc.